codedeploy_role_document.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Sid": "",
+    "Effect": "Allow",
+    "Principal": {
+      "Service": [
+        "codedeploy.us-east-1.amazonaws.com",
+        "codedeploy.eu-west-1.amazonaws.com",
+        "codedeploy.us-west-2.amazonaws.com",
```

Comment:
I think the install script for codedeployagent is kept not in the eu zone
and it needs to be downloaded ... not sure - need to check.


On 5 August 2015 at 10:18, Philip Potter <notifications@github.com> wrote:

> In codedeploy_role_document.json
> <https://github.com/openregister/deployment/pull/8#discussion_r36283019>:
>
> > @@ -0,0 +1,17 @@
> > +{
> > +  "Version": "2012-10-17",
> > +  "Statement": [{
> > +    "Sid": "",
> > +    "Effect": "Allow",
> > +    "Principal": {
> > +      "Service": [
> > +        "codedeploy.us-east-1.amazonaws.com",
> > +        "codedeploy.eu-west-1.amazonaws.com",
> > +        "codedeploy.us-west-2.amazonaws.com",
>
> yes, they're in eu-west-1. They're not in us-east-1 or us-west-2, so I'm
> not sure why these lines are required.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openregister/deployment/pull/8/files#r36283019>.
>



-- 
Thanks,

/Saqib